doc/nvim-treesitter.txt

```diff
@@ -166,4 +166,135 @@ Note: This is highly experimental, and folding can break on some types of
       edits. If you encounter such breakage, hiting `zx` should fix folding.
       In any case, feel free to open an issue with the reproducing steps.
 
+==============================================================================
+HIGHLIGHTS                                        *nvim-treesitter-highlights*
+
+TSError
```

Comment:
TSError: .. for syntax/parsing errors

Maybe you can add an introduction to this section. Especially, with an example how you could overwrite a highlighting. E.g. how to avoid highlighting of syntax errors.